webapp/channels/src/components/admin_console/color_setting.tsx

```diff
@@ -16,27 +16,27 @@ type Props = {
     disabled?: boolean;
 }
 
-export default class ColorSetting extends React.PureComponent<Props> {
-    private handleChange = (color: string) => {
-        if (this.props.onChange) {
-            this.props.onChange(this.props.id, color);
+const ColorSetting: React.FC<Props> = (props) => {
```

Comment:
we are following a different pattern, please use the same for consistency, here it would be:

```suggestion
const ColorSetting = ({id, label, helpText, value, onChange, disabled}: Props) => {
```